server/views/components/scroll-to-info/scroll-to-info.njk

```diff
@@ -0,0 +1,9 @@
+<a
+  href="#{{ model.elementId }}"
+  class="scroll-to-info js-scroll-to-info plain-link flex flex--column {{ 'scroll-to-info' | componentClasses(modifiers) if modifiers }} {% for class in data.extraClasses %}{{ class }} {% endfor %}"
+  data-track-click='{}'
+  data-component-name="Scroll to info"
+  data-component-id="{{ model.elementId }}">
+  {% icon 'actions/information', 'scroll to more information', data.extraInformationIconClasses %}
```

Comment:
"Scroll to" vs "See more information" or just "More information"? Not sure who to ask about what might be best?